.github/dependabot.yml

```diff
@@ -0,0 +1,22 @@
+version: 2
+updates:
+  - package-ecosystem: bundler
+    directory: "/"
+    schedule:
+      interval: daily
+      time: "07:00"
+    open-pull-requests-limit: 99
+    labels:
+      - dependencies
+    ignore:
+      - dependency-name: simplecov
+        versions:
+          - ">= 0.18"
```

Comment:
why are we ignoring this one?

Reply:
This gem isn't used, so has been removed.